resources/css/nav.css

```diff
@@ -111,7 +111,7 @@
   display: block;
   color: #000;
   line-height: 1.4em;
-  width: 150px;
+  width: 175px;
```

Comment:
```suggestion
  min-width: 175px;
  white-space: nowrap;
```

Only pushes the width if an item exceeds the maximum width whilst assuring none break lines.

Reply:
This is certainly more future-proof, but it does create an oddity where each column of the Games menu is a different width:
![image](https://user-images.githubusercontent.com/32680403/192648945-eed3ccbb-9fa8-431b-b4f0-575dc069e7fe.png)

3D0 Interactive Multiplayer appears in the second column, and has a more natural wrapping on the site currently so it didn't stand out.

Reply:
That's fair, fixed 175px makes more sense then to keep that intact.